### PR TITLE
shallow render error messages now only show components under test

### LINKED ISSUE #218: Error message for shallow render contains
Currently it shows you the entire render html when this errors.

For example:

```
function Bar() {
  return (
    <div>
      <div className="in-bar" />
    </div>
  );
}
```

```
function Foo() {
  return (
    <div>
      <Bar attr="12" />
    </div>
  );
}
```

```
expect(shallow(<Foo>)).to.contain(<Bar attr="13" />);
```

when this errors it does a .html() on the wrapper and would show:

```
Failure: Expected <Foo /> to contain <Bar attr="13" />

<div>
    <div>
      <div className="in-bar" />
    </div>
</div>
```
This makes it extremely hard to understand exactly why the test is failing. 

Would anyone be against adding an overload to do a .debug() instead so the following would be output.  Or better yet, always use debug for shallow wrappers. 

So the following would be shown:
```
Failure: Expected <Foo /> to contain <Bar attr="13" />

<div>
    <Bar attr="12">
</div>
```

Thoughts?

### PR DESCRIPTION
**Pull request by armsteadj1**
(the first level components)

Fixes #218.

### CODE REVIEW

File: src/ShallowTestWrapper.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ export default class ShallowTestWrapper extends TestWrapper {
   }
 
   html () {
-    return this.wrapper.html()
+    return this.wrapper.debug().replace(/\n(\s*)/g, '')
```
Comment (ljharb) [inline src/ShallowTestWrapper.js]: what's the `.replace` for?

Comment (armsteadj1) [reply src/ShallowTestWrapper.js]: When I updated it to debug() a few of the html() tests were failing:

AssertionError: expected the node in <Fixture /> to be '<span id="child">Test</span>', but it was '<span id="child">\n  Test\n</span>'      

This change removed that extra padding and the \n's to get them to match what they use to be. I took the regex from the jest-enzyme link I posted in the #218.

Let me know if you'd rather it be the other way and I can update those tests and remove the regex!

**Comment by armsteadj1** [comment]
How does one get a pull request merged in and pushed out as a new version to npm? 😄

**Comment by gunnari-turo** [comment]
Any update on getting this merged and a new version created @ayrton or @ljharb?

**Pull request merged**